src/components/ticket.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
```

Comment:
I suggest writing the name of React Components capitalized.
So the name of this file would be `src/components/Ticket.js`